src/factoid.coffee

```diff
@@ -23,9 +24,11 @@
 
 class Factoids
   constructor: (@robot) ->
-    @robot.brain.on 'loaded', =>
-      @cache = @robot.brain.data.factoids
-      @cache = {} unless @cache
+    storageLoaded = =>
+      @cache = @robot.brain.data.factoids ||= {}
+      @robot.logger.debug "Factoids Data Loaded: " + JSON.stringify(@cache, null, 2)
+    @robot.brain.on 'loaded', storageLoaded
+    storageLoaded() # just in case storage was loaded before we got here
```

Comment:
looks like it was added here
https://github.com/ajacksified/hubot-plusplus/commit/6729995a73f9ac809c43fb47fab483ce1644be31

And the issue is

https://github.com/ajacksified/hubot-plusplus/issues/13